package.json

```diff
@@ -23,40 +23,40 @@
     "test:tofile": "npm run build:babel && TZ='GMT' mocha --no-colors -R spec \"test/**/*.js\" 2>&1 | tee test-report.log"
   },
   "devDependencies": {
-    "@babel/cli": "^7.4.3",
-    "@babel/core": "^7.4.3",
+    "@babel/cli": "^7.4.4",
+    "@babel/core": "^7.4.5",
     "@babel/plugin-proposal-json-strings": "^7.0.0",
-    "@babel/plugin-transform-runtime": "^7.4.3",
+    "@babel/plugin-transform-runtime": "^7.4.4",
     "@babel/plugin-transform-strict-mode": "^7.0.0",
-    "@babel/preset-env": "^7.4.3",
+    "@babel/preset-env": "^7.4.5",
     "@babel/preset-react": "^7.0.0",
```

Comment:
do we need react transform here?  There is only one component render in each file that can be replaced with `React.createElement`

Reply:
Build time dependency - doesnt matter